app/graphql/resolvers.ts

```diff
@@ -223,6 +230,15 @@ export const resolvers: Resolvers = {
   RawObservation: GraphQLJSONObject,
   Query,
   DataCube,
+  DataCubeTheme: {
+    label: async (parent, _, { loaders }) => {
+      if (!parent.label) {
```

Comment:
Is it ever the case that `parent.label` is preloaded at the parent query?

Reply:
Yes, when the themes are all loaded (to be displayed in the left panel).